### PR TITLE
Assignment 4 - Ian Kaiserman

### PR DESCRIPTION
**Pull request by Afterlyte**
Paired with @TannerStephenson

https://kind-stone-01ab5711e.3.azurestaticapps.net/

Still a work in progress, the bulk of the work should be done but we're still working on the overall Axios calls and linking the back-end to the front-end. Also deployment may be broken currently but it should run as-is locally.

EDIT: Entire site is now working as intended, but only locally right now. We're investigating issues with the API changes not properly being sent to the Azure SQL DB but it might not be fixed by the due date, so run it locally for correct results for now.

### CODE REVIEW

File: wordle-web/src/components/ScoreDialog.vue [hunk 1]
```diff
@@ -1,8 +1,8 @@
 <template>
   <v-dialog :model-value="modelValue" @update:model-value="close" :max-width="300">
     <v-card>
-      <div class="d-flex justify-center pa-2 bg-primary text-h6">
-        You {{ gameResult?.wasGameWon ? 'Win' : 'Loose' }}
+      <div class="d-flex justify-center pa-2 bg-cyan text-h6">
```
Comment (schemcharger) [inline wordle-web/src/components/ScoreDialog.vue:4]: I like the use of cyan to make the buttons stand out

File: wordle-web/src/views/InstructionsView.vue [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  <v-card class="mx-auto mt-9" width="475px" color="yellow-lighten-1">
+    <v-card color="cyan-accent-3">
+      <v-card-text class="text-center" style="font-size: xx-large; font-weight: 600">
+        How to Play
+      </v-card-text>
+    </v-card>
+    <v-card-text style="font-size: large; font-weight: 620">
+      1. Guess the five-letter word by entering your guess in the text box and clicking the "Guess"
+      button.
+      <br /><br />2. You have six guesses to get the word right. <br /><br />3. If you guess a
```
Comment (schemcharger) [inline wordle-web/src/views/InstructionsView.vue:11]: I would recommend breaking it into new lines in the code to improve readability

**Review by schemcharger — COMMENTED** [review]
Overall, It looks good so far.

**Comment by arose-13** [comment]
Sorry about this being late, but I'm gonna review this.

**Review by arose-13 — COMMENTED** [review]
# Features
* Make all pages responsive ✅
  * 1080p desktop ✅
  * iPad Air ✅
  * Samsung Galaxy S20 Ultra Super Mega Excellent ✅
  * IPhone SE ✅
* Create a improved landing page and site theme (Background, make it look attractive to play) ✅
* Theme should extend all pages ✅
* Add an instructions page that is available via the hamburger menu that details the features of the game and how to play ✅
* Create a new Wordle game page that reuses components but will provide the same word for a specific day. Everyone going to the page that day will get the same word. The page should indicate Wordle of the Day or something like that ✅
* Create a page that lists the last 10 daily words with statistics. Number of plays, Average score, Average time. (just list the date, don't show the word silly) ❌
  * It should indicate if the user has played that day ❌
 
 A few points:
 * I really like the theme! It's awesome. However, the light/dark themes don't do much now.
 * Your page is responsive but can appear a little squished on the phone screens. Maybe try increasing the width of the keyboard and gameboard when below certain screen widths.
 * The stats page isn't working for me.

Other than that, it looks great!

**Comment by TannerStephenson** [comment]
> Paired with @TannerStephenson
> 
> https://kind-stone-01ab5711e.3.azurestaticapps.net/
> 
> Still a work in progress, the bulk of the work should be done but we're still working on the overall Axios calls and linking the back-end to the front-end. Also deployment may be broken currently but it should run as-is locally.
> 
> EDIT: Entire site is now working as intended, but only locally right now. We're investigating issues with the API changes not properly being sent to the Azure SQL DB but it might not be fixed by the due date, so run it locally for correct results for now.

Interesting thing I found, our deployed website's stats page actually works but only in incognito? If you don't use incognito on the deployed website, chrome will block the requests. Unsure on why this is happening.

File: wordle-web/src/App.vue [hunk 3]
```diff
@@ -1,46 +1,61 @@
...
+                <v-list-item-title>
+                  <RouterLink to="/leaderboard">Leaderboard</RouterLink>
+                </v-list-item-title>
+              </v-list-item>
+              <v-list-item>
+                <RouterLink to="/instructions">Instructions</RouterLink>
+              </v-list-item>
+              <v-list-item>
+                <v-list-item-title
+                  ><RouterLink to="/stats">Statistics</RouterLink></v-list-item-title
```
Comment (mmwoodfo) [inline wordle-web/src/App.vue:44]:
```suggestion
                  > <RouterLink to="/stats"> Statistics </RouterLink> </v-list-item-title
```
If you add spaces between these, your linter will format the code a bit nicer!

**Review by mmwoodfo — COMMENTED** [review]
I played todays game - but doesn't look like it's registering it correctly
image


Also, looks like you have a few issues when resizing the screen with your dropdown nav and some pages
image

image

**Pull request closed without merging**